client/landing/stepper/declarative-flow/internals/components/site-preview/preview-toolbar.scss

```diff
@@ -34,19 +34,20 @@
 
 // Need the extra specificity here to override core
 .button.preview-toolbar__button {
```

Comment:
Maybe setting the `color` + adding `fill: currentcolor`  is a solution that covers any text added to the button?

Reply:
Yep, doesn't hurt to cover it just in case. Done in https://github.com/Automattic/wp-calypso/pull/100057/commits/2bc103dcf008301d240e54e427e0e287f479fdc0.